docs/examples/Example_data_fusion_from_sensors.py

```diff
@@ -0,0 +1,506 @@
+
+# initiate the tracks
+ukf_tracks = set()
+ekf_tracks = set()
+pf_tracks = set()
+truths = set()
+
+# list for all detections
+full_detections = []
+
+# loop over the various time-steps
+for t in range(number_of_steps):
+    detections_1 = next(g1)
+    s1_detections.extend(detections_1[1])
+    detections_2 = next(g2)
+    s2_detections.extend(detections_2[1])
+    full_detections.extend(detections_1[1])
+    full_detections.extend(detections_2[1])
+
+    for detections in [detections_1, detections_2]:
+
+        # Run the Kalman tracker
+        UKF_tracker.detector = DummyDetector(current=detections)
+        UKF_tracker.__iter__()
+        _, tracks = next(UKF_tracker)
+        ukf_tracks.update(tracks)
+        del tracks
+
+        # Run the extended Kalman filter
+        EKF_tracker.detector = DummyDetector(current=detections)
+        EKF_tracker.__iter__()
+        _, tracks = next(EKF_tracker)
+        ekf_tracks.update(tracks)
+        del tracks
+
+        # Run the Particle Tracker
+        PF_tracker.detector = DummyDetector(current=detections)
+        PF_tracker.__iter__()
+        _, tracks = next(PF_tracker)
+        pf_tracks.update(tracks)
+
+metric_manager.add_data({'UKF_tracks': ukf_tracks}, overwrite=False)
+metric_manager.add_data({'PF_tracks': pf_tracks}, overwrite=False)
+metric_manager.add_data({'EKF_tracks': ekf_tracks}, overwrite=False)
+
+truths = set(ground_truth_simulator.groundtruth_paths)
+metric_manager.add_data({'truths': truths,
+                         'detections': full_detections}, overwrite=False)
+
+# %%
+# 4) Plot the tracks and the track performances
+# ---------------------------------------------
+# We have obtained the tracks and the ground truths
+# from the various trackers. It is time
+# to visualise the tracks and load the metric manager
+# to evaluate the performances.
+
+plotter = Plotterly()
+plotter.plot_measurements(s1_detections, [0, 2], marker= dict(color='blue'),
+                         measurements_label='Radar 1 detections')
+plotter.plot_measurements(s2_detections, [0, 2], marker= dict(color='cyan'),
+                         measurements_label='Radar 2 detections')
+plotter.plot_tracks(ukf_tracks, [0, 2], line= dict(color='grey'), track_label='UKF tracks')
+plotter.plot_tracks(ekf_tracks, [0, 2], line= dict(color='black'), track_label='EKF tracks')
+plotter.plot_tracks(pf_tracks, [0, 2], particle=False, line= dict(color='red'),
+                    track_label='PF tracks')
+plotter.plot_ground_truths(truths, [0, 2])
+plotter.plot_sensors(sensor1_platform, [0, 1], marker=dict(color='black', symbol='129', size=15),
+                     sensor_label='Fixed Platform')
+plotter.plot_ground_truths(sensor2_platform, [0, 2], marker=dict(color='orange', symbol='cross',
+                                                           size=25),
+                     truths_label='Moving Platform')
+plotter.fig
+
+# Loaded the plotter for the various metrics.
```

Comment:
The plot of the actual tracks is not rendering in the documentation. Try splitting this into two cells so that the first ends with `plotter.fig` and the metrics plotting is in a separate code cell.

```suggestion
plotter.fig

# %%
# Plot the metrics

# Loaded the plotter for the various metrics.
```